app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -24,6 +24,54 @@ def from_code
         render :show
       end
 
+      def start
+        @lobby = Lobby.find(params[:id])
+
+        if @lobby.update(status: :in_progress)
+          Pusher.trigger(@lobby.code, Lobby::LOBBY_START, {})
+          render :show
+        else
+          render 'api/v1/model_errors', locals: { errors: @lobby.errors }, status: :unprocessable_entity
+        end
+      end
+
+      def answer
+        lobby = Lobby.find(params[:id])
+        player = Player.find_by(id: params[:player_id])
+        return head :not_found if player.nil?
```

Comment:
You can also use `Player.find(params[:player_id]` which will raise an `ActiveRecord::NotFound` exception, which will in return make the API respond with a `not_found` status (if I'm not wrong 😅)